docs/openj9_defaults.md

```diff
@@ -81,15 +81,16 @@ The last 2 columns show whether the default setting can be changed by a command-
-|Stack size for OS Threads **(64-bit)**. Use `-Xmso<size>`           |256 KB    |256 KB    |256 KB           |256 KB           |1 MB    |<i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>| <i class="fa fa-times" aria-hidden="true"></i><span class="sr-only">no</span> |
+|Stack size for OS Threads **(64-bit)**. Use `-Xmso<size>`           |256 KB    |256 KB    |256 KB           |256 KB           |1 MB min (See **Note 1**)   |<i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>| <i class="fa fa-times" aria-hidden="true"></i><span class="sr-only">no</span> |
 |Initial heap size. Use `-Xms<size>`                           |8 MB      |8 MB      |8 MB             |8 MB             |8 MB      |<i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>| <i class="fa fa-times" aria-hidden="true"></i><span class="sr-only">no</span> |
-|Maximum Java heap size. Use `-Xmx<size>`                      |See **Notes**|See **Notes**|See **Notes**|See **Notes**|See **Notes**|<i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>| <i class="fa fa-times" aria-hidden="true"></i><span class="sr-only">no</span> |
+|Maximum Java heap size. Use `-Xmx<size>`                      |See **Note 2**|See **Note 2**|See **Note 2**|See **Note 2**|See **Note 2**|<i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>| <i class="fa fa-times" aria-hidden="true"></i><span class="sr-only">no</span> |
 |Page size for the Java object heap and code cache (For restrictions, see the `-Xlp:codecache` and `-Xlp:objectheap` options).|Operating system default                                                           |Architecture: x86: 2MB, IBM Z&reg;: 1MB, Other architectures: Operating system default |4 KB        | Operating system default | 1M pageable | <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>  | <i class="fa fa-times" aria-hidden="true"></i><span class="sr-only">no</span>                                                                                                                                                                                                                                                                            
 
-
 <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Notes:**
 
-The default value of `-Xmx` depends on the version of Java.
+1. On z/OS 64-bit, [`-Xmso`](xmso.md) has a 1 MB minimum value, to match the minimum stack space provided by the OS. Setting a value smaller than 1 MB is ignored.
+
+2. The default value of [`-Xmx`](xms.md) depends on the version of Java.
 
 - ![Start of content that applies only to Java 8 (LTS)](cr/java8.png) The value is half the available memory with a minimum of 16MB and a maximum of 512 MB. ![End of content that applies only to Java 8 (LTS)](cr/java_close_lts.png)
 
```

Comment:
I don't think this topic needs changing at all as the default value remains the same - this topic is just a reference topic to tell users what the defaults are for the whole VM. Sue suggested changing xmso.md, which I agree with - that's where users should go to find out about the behaviour of the option. In xmso.md, there is this sentence:

> Default values vary by platform. See Default settings for the OpenJ9 VM.

I suggest adding the following after that sentence:

> `<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note:** On 64-bit z/OS, the default size is the minimum that can be allocated by the operating system. So if you set a value that is smaller, that value is ignored by the VM.`

(I've avoided mentioning the actual value here, so that it doesn't become out of date if the openj9_defaults.md topic changes in future).

Reply:
Done. I kept the link for -Xmx and aligned the columns in the text, just to make it easier when making future changes.

Reply:
lgtm, thanks